src/main/java/seedu/address/logic/commands/CountryFilterCommand.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.CountryMatchesInputCountryPredicate;
+
+public class CountryFilterCommand extends Command {
```

Comment:
There needs to be a javadoc comment here. (All public classes should have javadoc according to the textbook)

Reply:
Added